scripts/check_type_hint_coverage.py

```diff
@@ -4,7 +4,7 @@
 from typing import Dict, List, Optional
 
 TYPE_HINT_ERROR_THRESHOLD: int = (
-    2471  # This number is to be reduced as we annotate more functions!
+    208  # This number is to be reduced as we annotate more functions!
```

Comment:
Wow, that a big reduction!

Reply:
Only because it's now totally ignoring a bunch of modules 😄 .
Temporarily taking a step back in order to take many steps forward.